.github/workflows/django-base-image.yml

```diff
@@ -0,0 +1,28 @@
+on:
```

Comment:
This pipeline is to keep our testing image up-to-date. Each night it will execute and will get the latest python versions

Reply:
Our test matrix in python about minor version i.e 3.8 3.9 etc. We are not testing for patch version i.e. 3.8.15 3.8.16 
Therefore patch version are the same for us
We are just installin the latest up-to-date patch version and it is good to have I think. It shouldn't affect us. If it is, then we definetely need to fix it